WTWSMS/decisions/WtWSMS_religious_decisions.txt

```diff
@@ -429,17 +431,36 @@ decisions = {
 		
 		effect = {
 			give_nickname = nick_with_immortal_soul
-			activate_title = { title = d_priscillianism status = yes }
+			set_global_flag = veggiepriest
+			activate_title = { title = k_priscillianism status = yes }
 			hidden_tooltip = {
-				create_random_priest = {
+				create_character = {
+					name = "Priscillian"
 					dynasty = none
-					random_traits = yes
+					trait = humble
+					trait = vegetarian
+					trait = zealous
+					trait = mystic
+					race = egyptian_arabic
+					religion = priscillianism
+					culture = ROOT
+					female = no
+					attributes = {
+						martial = 8
+						intrigue = 10
+						learning = 10
+					}
+					historical = yes
 				}
 				new_character = {
-					d_priscillianism = {
+					k_priscillianism = {
```

Comment:
No, the version you set requires Priscillianism to be the dominant Christian faith on the earth. That is inaccurate to the Priscillian theology and mindset. My original version of the Decision was far better.

Similarity does not matter, but balance does. Having the Prisicllian Pope a king-tier position makes him harder to control, like the Bishop of Rome. Priscillian clergy were fiercely independent, hence why they hated the Papacy trying to control them in the first place. In addition, the Priscillian religion, being so similar to the Catholics in terms of the primacy and power of the clergy, has extrmely powerful abilities such as Grant Claim, Invasion, and Investiture.